clar2wasm/src/words/responses.rs

```diff
@@ -61,34 +71,17 @@ impl Word for IsErr {
         _expr: &SymbolicExpression,
         args: &[SymbolicExpression],
     ) -> Result<(), GeneratorError> {
-        let res = args.get_expr(0)?;
-        generator.traverse_expr(builder, res)?;
-        // there is a response type on top of the stack.
-
-        // Get the type of the response expression
-        let ty = generator
-            .get_expr_type(res)
-            .expect("input expression must be typed")
-            .clone();
-
-        let (ok_ty, err_ty) = if let TypeSignature::ResponseType(types) = &ty {
-            &**types
-        } else {
-            panic!("Expected a Response type. Found: {:?}", ty);
+        match traverse_response(generator, builder, args) {
+            Ok(_) => {
+                // Add one to stack
+                // and proceed with a XOR operation
+                // to invert the indicator value
+                builder.i32_const(1);
+                builder.binop(BinaryOp::I32Xor);
+            }
+            Err(e) => return Err(e),
         };
```

Comment:
```suggestion
        traverse_response(generator, builder, args)?;

        // Add one to stack
        // and proceed with a XOR operation
        // to invert the indicator value
        builder.i32_const(1).binop(BinaryOp::I32Xor);
```

Reply:
That would raise a warning. But we can mute it with a silent binding. What you think?

Reply:
Ah, I see Anthony suggested the same thing that I just commented on. Why wouldn't you want that error to be returned from this function?

Reply:
Why would it raise a warning? the return type of `traverse_response(...)?` is `()`, right?